src/zcl_abapgit_version.clas.abap

```diff
@@ -242,4 +249,27 @@ CLASS zcl_abapgit_version IMPLEMENTATION.
     rv_version = lv_major * 1000000 + lv_minor * 1000 + lv_release.
 
   ENDMETHOD.
+
+  METHOD get_version_constant_value.
+    DATA: lv_version_class     TYPE string,
+          lv_version_component TYPE string.
+    FIELD-SYMBOLS: <lv_version> TYPE string.
+
+    IF iv_version_constant NP '*=>*'.
+      zcx_abapgit_exception=>raise( 'Version constant needs to use the format CLASS=>CONSTANT' ).
```

Comment:
guess it will also work with interfaces?

Reply:
Yes it does (see change in abapGit's own .abapGit.xml . I couldn't think of a good way to account for this in the message and also the UI hint. `OBJECT` to me rather refers to instances and `{ CLASS / INTERFACE }=>CONSTANT` might just confuse. Maybe `CLASS=>CONSTANT / INTERFACE=>CONSTANT` would be better.

Reply:
Well actually attributes will also work. So `COMPONENT` would be more correct than `CONSTANT`. Attributes will also call `class_constructor` though (do constants do that as well?) which means code execution of the (active) repo code :/.

Reply:
class constants do not trigger the class constructor. reading class attributes does (on 754)